src/views/Curators/Curator.vue

```diff
@@ -0,0 +1,534 @@
+                value: "creator"
+              }
+            ]
+          },
+          search1: "",
+          search2: "",
+          search3: "",
+          search5: "",
+          search6: "",
+          search7: ""
+        }
+      },
+      computed: {
+        ...mapState('users', ['user', "messages"]),
+      },
+      created() {
+        this.$nextTick(function () {
+          this.recordType = recordTypes;
+        });
+      },
+      async mounted() {
+          await this.getUser();
+          if (this.messages()["getUser"].error){
+            this.setError({field:"login", message:"You've been logged out automatically"});
+            this.$router.push({path: "/accounts/login"})
+          }
+          client.setHeader(this.user().credentials.token);
+          let data = await client.executeQuery(getCurationRecords);
+          this.allDataCuration = data["curationSummary"];
+          client.initalizeHeader();
+          this.prepareData();
+      },
+      methods: {
+          ...mapActions('users', ['getUser', 'setError']),
+          prepareData(){
+            let _module = this;
+            _module.prepareApprovalRequired(_module.allDataCuration);
+            _module.prepareMaintenanceRequests(_module.allDataCuration);
+            _module.prepareRecordsInCuration(_module.allDataCuration);
+            _module.prepareRecordsWithoutDois(_module.allDataCuration);
+            _module.prepareHiddenRecords(_module.allDataCuration);
+            _module.prepareRecordsCuratorCreationsLastWeek(_module.allDataCuration);
+          },
+          prepareApprovalRequired(dataCuration){
+            let _module = this;
+            let userRecords = dataCuration['approvalsRequired'];
+            userRecords.forEach(item => {
+              item['fairsharingRecords'].forEach(rec => {
+                let object = {};
+                object.updatedAt = rec.updatedAt;
+                object.curator = item['username'];
+                object.recordName = rec.name;
+                object.id = rec.id;
+                object.type = rec.type;
+                if (rec['lastEditor'] != null){
+                  object.lastEditor = rec['lastEditor'].username+' ('+rec['lastEditor'].id+')';
+                }else{
+                  object.lastEditor = "unknown"
+                }
+
+                _module.approvalRequired.push(object);
+              });
+            });
+          },
+          prepareMaintenanceRequests(dataCuration){
+            let _module = this;
+            let requests = dataCuration['pendingMaintenanceRequests'];
+            requests.forEach(item => {
+              let object = {};
+              object.createdAt = item.createdAt;
+              object.recordNameID = item['fairsharingRecord'].name+' ('+item['fairsharingRecord'].id+')';
+              object.userNameID = item['user'].username+' ('+item['user'].id+')';
+              _module.maintenanceRequests.push(object);
```

Comment:
You don't really have to rename `this`. Assigning this to a new variable only makes sense if you use that variable inside an anonymous function if that function does not use arrow functions. Here you are using arrow functions so you don't have to alias this. Simple rule. If you write an arrow function the `this` inside the arrow function is exactly the same as the `this` outside.

```suggestion
            let requests = dataCuration['pendingMaintenanceRequests'];
            requests.forEach(item => {
              let object = {};
              object.createdAt = item.createdAt;
              object.recordNameID = item['fairsharingRecord'].name+' ('+item['fairsharingRecord'].id+')';
              object.userNameID = item['user'].username+' ('+item['user'].id+')';
              this.maintenanceRequests.push(object);
```